netket/driver/abstract_variational_driver.py

```diff
@@ -232,7 +232,7 @@ def run(
             )
 
         # Log only non-root nodes
-        if self._mynode == 0:
+        if self._mynode == 0 and jax.process_index() == 0:
```

Comment:
can we just define something like self._is_root = self._mynode == 0 and jax.process_index() == 0 ?

Reply:
done